fud2/fud-core/src/run.rs

```diff
@@ -383,7 +383,15 @@ impl<'a> Run<'a> {
                 }
             }) {
                 let stdout_files =
-                    std::fs::File::open(self.plan.workdir.join(filename))?;
+                    std::fs::File::open(self.plan.workdir.join(filename))
+                        .map_err(|e| if let std::io::ErrorKind::NotFound = e.kind() {
+                            std::io::Error::new(
+                                e.kind(),
+                                format!("{}\nHint: Check ops actually generate all of their targets.", e)
+                            )
+                        } else {
+                            e
+                })?;
```

Comment:
Just because this took me a little while to grok what was going on, maybe a comment would be in order here. Something like this:

> The output file we're emitting to stdout should exist. If it doesn't, some op didn't produce the output file it claimed to.